module/src/rotation.js

```diff
@@ -138,7 +138,7 @@ async function rotateViaRotation(deltaX, deltaY, document, update){
 
 
 async function rotateTokenOnPreUpdate(token_document, change, options, userId) {
-    const token_data = token_document.data
+    const token_data = token_document
```

Comment:
I was just working on something like this yesterday! :)

I made almost the exact same changes as you, then realized that this change set doesn't function correctly in v9.
An error is thrown trying to get `token_data.flags` in `shouldRotate`

Did something like this instead:
`const token_data = isV10() ? token_document : token_document.data;`